openedx/core/djangoapps/user_api/views.py

```diff
@@ -1036,3 +1037,17 @@ def post(self, request):
         email_opt_in = request.data['email_opt_in'].lower() == 'true'
         update_email_opt_in(request.user, org, email_opt_in)
         return HttpResponse(status=status.HTTP_200_OK)
+
+
+class CountryTimeZoneListView(generics.ListAPIView):
+    """
+    DRF class for listing common time zones for given country
+    or all time zones if no country given
```

Comment:
Fill out this docstring a little more.  It should specify the parameters that can be passed, and explain that the country code is in two-letter ISO-3166 format, and is case insensitive.  See some other API views (such as openedx/core/djangoapps/profile_images/views.py) for details and formatting.